scipy/spatial/distance.py

```diff
@@ -1373,6 +1373,10 @@ def dice(u, v, w=None):
     dice : double
         The Dice dissimilarity between 1-D arrays `u` and `v`.
 
+   Note: This function computes the Dice dissimilarity index. To compute the 
```

Comment:
```suggestion
    Notes
    -----
    This function computes the Dice dissimilarity index. To compute the 
```

Reply:
Thanks @tirthasheshpatel. I have accepted your commit suggestion, but it's still failing. Do you know why or how to fix this?